_includes/runs-new.md

```diff
@@ -13,30 +13,41 @@ instrument Run, and contains one or more lanes. Each Illumina lane, PacBio SMRT
 cell, or Oxford Nanopore flow cell is loaded with exactly one _Pool_.  Runs and
 Containers can be associated as soon as the Run and Container have both been created.
 
-## 3.1 Create a Container
+## {{include.section}}.1 Create a Container
 
-1. On the _Sequencing Containers_ page, select the Illumina platform and click the
-   _Create Flowcell_ button near the top left of the table.
-1. Select _{{ site.flowcell }}_ from the dialog. Only active (non-archived) models are
+1. On the _Sequencing Containers_ page, select the {{include.platform_type}} platform and
+   click the _Add Flow Cell_ button near the top left of the table.
+1. Select _{{ include.platform }}_ from the dialog.
+1. Select _{{ include.flowcell }}_ from the dialog. Only active (non-archived) models are
    available.
 1. Enter the name of your project as the serial number.
+{% if include.platform_type == 'Oxford Nanopore' %}
+1. Select a _Pore Version_.
+1. Select a _Received Date_. The _Returned Date_ is the date that the flow cell was returned
+to be recycled. This should be left blank for now, and may be filled out later.
+{% endif %}
 1. Click _Save_ in the upper right corner of the page.
 
-## 3.2 Create a Run
+## {{include.section}}.2 Create a Run
 
-1. On the _Sequencer Runs_ page, select the Illumina platform and click the 
-   _Add Illumina Run_ button near the top left corner of the table.
-1. Select _{{ site.sequencer }} ({{ site.platform }})_ from the sequencers list. Only
+1. On the _Sequencer Runs_ page, select the {{ include.platform_type }} platform and click the 
+   _Add {{include.platform_type}} Run_ button near the top left corner of the table.
+1. Select _{{ include.sequencer }} ({{ include.platform }})_ from the sequencers list. Only
    active (non-retired) sequencers are available.
 1. Enter a unique and memorable _Alias_ for your run.
-1. Select _Sequencing Parameters_ `{{ site.seq_params }}`, near the top of the Run
+1. Select _Sequencing Parameters_ `{{ include.seq_params }}`, near the top of the Run
```

Comment:
"near the top of the run section"? On the page it's halfway down, and on the dropdown list it's near the end. Not sure what this is meant to refer to